CHANGES.rst

```diff
@@ -67,6 +67,8 @@ New Features
   - Added the injection of EntryPoints into astropy.modeling.fitting if
     they inherit from Fitters class. [#5241]
 
+  - Added ``Planar2D`` functional model.
```

Comment:
You need to include this PR number in the change log. See other entries for formatting.